solutions/instances/variables.tf

```diff
@@ -235,6 +248,22 @@ variable "enable_metrics_routing_to_cloud_monitoring" {
   default     = true
 }
 
+variable "metrics_router_settings" {
+  type = object({
+    default_targets = optional(list(object({
+      id = string
+    })), [])
+    permitted_target_regions  = optional(list(string))
+    primary_metadata_region   = optional(string)
+    backup_metadata_region    = optional(string, "us-east")
+    private_api_endpoint_only = optional(bool, false)
+  })
+  description = "Global settings for Metrics Routing. [Learn more](https://github.com/terraform-ibm-modules/terraform-ibm-observability-da/blob/main/solutions/instances/DA-types.md#metrics-router-settings-)"
+  default = {
+    primary_metadata_region  = "eu-de"
+    permitted_target_regions = []
+  }
```

Comment:
Should we really be setting a default value here? Should the default no be `null` but allow users to opt in if they want to any of the settings? Will metrics routing work in a brand new fresh account if we default this to `null` ?